app/client/platforms/bedrock.ts

```
@@ -0,0 +1,417 @@
+
+        while (true) {
+          const { done, value } = await reader.read();
+          if (done) break;
+
+          // Convert the chunk to text and add to buffer
+          const chunk = new TextDecoder().decode(value);
+          buffer += chunk;
+
+          // Process complete messages from buffer
+          let newlineIndex;
+          while ((newlineIndex = buffer.indexOf("\n")) !== -1) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)